src/braket/circuits/measure.py

```diff
@@ -0,0 +1,99 @@
+# Copyright Amazon.com Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"). You
+# may not use this file except in compliance with the License. A copy of
+# the License is located at
+#
+#     http://aws.amazon.com/apache2.0/
+#
+# or in the "license" file accompanying this file. This file is
+# distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF
+# ANY KIND, either express or implied. See the License for the specific
+# language governing permissions and limitations under the License.
+
+from __future__ import annotations
+
+from typing import Any
+
+from braket.circuits.quantum_operator import QuantumOperator
+from braket.circuits.serialization import (
+    IRType,
+    OpenQASMSerializationProperties,
+    SerializationProperties,
+)
+from braket.registers.qubit_set import QubitSet
+
+
+class Measure(QuantumOperator):
+    """Class `Measure` represents a measure operation on targeted qubits"""
+
+    def __init__(self, **kwargs):
+        """Inits a `Measure`.
+
+        Raises:
+            ValueError: `qubit_count` is less than 1, `ascii_symbols` are `None`, or
+                `ascii_symbols` length != `qubit_count`
+        """
+        super().__init__(qubit_count=1, ascii_symbols=["M"])
+        self._target_index = kwargs.get("index")
+
+    @property
+    def ascii_symbols(self) -> tuple[str, ...]:
```

Comment:
Since this is a 1 qubit operation, wouldn't `ascii_symbols` be just a `tuple[str]`?

Reply:
Yes, good catch!